packages/core/src/fhir-to-cda/cda-types/shared-types.ts

```diff
@@ -91,6 +94,11 @@ export type CDACodeCE = {
   [displayNameAttribute]?: string;
 };
 
+export type CDAValueST = {
+  [xsiTypeAttribute]?: string;
```

Comment:
example of a place that could be defined differently if we change attribute name:

```
export type CDACodeCE = {
  _codeAttribute?: string;
  _codeSystemAttribute?: string;
  _codeSystemNameAttribute?: string;
  _displayNameAttribute?: string;
};
```

Reply:
I dont think this works. If you define it with the underscore, it doesn't use it as a variable anymore. So in this case, the actual attributes will be: 
`CDACodeCE._codeAttribute` instead of `CDACodeCE._code` as defined in the `codeAttribute` variable.

Reply:
Renamed the variables to also have the underscore. Resolved